corehq/apps/data_interfaces/migrations/0036_populate_dedupe_match_values.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.23 on 2023-12-04 22:44
+from django.db import migrations, transaction
+from corehq.apps.data_interfaces.models import CaseDuplicate
+from corehq.form_processor.models.cases import CommCareCase
```

Comment:
As we discussed yesterday, these should not be imported in a migration. Instead, minimal implementations of the parts that are needed should be included directly in this file.